### PR TITLE
Integrate django rest framework

### LINKED ISSUE #2: Should integrate Django REST Framework (DRF)
We should use DRF to expose a REST API that our react frontend can use to do things like enumerate blog posts, render blog post details page, and upvote

### PR DESCRIPTION
**Pull request by luisnaranjo733**
We should use DRF to expose a REST API that our react frontend can use to do things like enumerate blog posts, render blog post details page, and upvote

This PR integrate the DRF by following their tutorial
https://www.django-rest-framework.org/tutorial/quickstart

Closes #2

### CODE REVIEW

File: backend/blog/urls.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 from . import views
 
-app_name = 'blog'
```
Comment (TaraWilson17) [inline backend/blog/urls.py:5]: Why did you remove the `app_name`?

Comment (luisnaranjo733) [reply backend/blog/urls.py:5]: oops!😱

Comment (luisnaranjo733) [reply backend/blog/urls.py:5]: All fixed 🎉
